src/teleop/download_map/download_map.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+import zipfile
+import json
+import boto3
+import subprocess
+
+
+# If System has issues with number of file watcher when using GUI with downloaded map
+# Run "sudo sysctl -w fs.inotify.max_user_watches=100000"
+def main():
+    result = subprocess.Popen(
+        ["rospack", "find", "mrover"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )  # runs rospack find mrover to get directory
+    m_bytes, err = result.communicate()
+    m = m_bytes.decode("utf-8")  # convert byte to string type
+    m = m[: len(m) - 1]  # the old m includes the '\n' and this gets rid of that
+    mrover = Path(m)  # load directory into path
+    path_exists = Path.exists(mrover / "src" / "teleop" / "download_map" / "keys.json")
+    if not path_exists:
+        out = {"accessKey": "", "secretKey": ""}
+        f = open(mrover / "src" / "teleop" / "download_map" / "keys.json", "w")
+        json.dump(out, f, indent=4)
+        print("Access Keys not found, input keys into keys.json")
+        exit(0)
```

Comment:
I'm slightly OCD about code formatting but I always think it's nice to include an extra newline after chunks of code that seem like a single thought or paragraph (like after this exit statement, and generally throughout this script) to make things easier to read